examples/remix-auth-auth0/app/constants/index.ts

```diff
@@ -0,0 +1,7 @@
+export const AUTH0_RETURN_TO_URL = process.env.AUTH0_RETURN_TO_URL!;
```

Comment:
Maybe rename it as .server.ts? Since it uses env variables it should never be sent to the client

Reply:
I got it. I rename it.